### PR TITLE
Updated the Project.toml

### PR DESCRIPTION
**Pull request by VarLad**

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -18,7 +18,7 @@ ShaderAbstractions = "65257c39-d410-5151-9873-9b3e5be5013e"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 
 [compat]
-AbstractPlotting = "0.15.4"
+AbstractPlotting = "0.15"
```
Comment (SimonDanisch) [inline Project.toml]: This actually needs to be 0.15.4

Comment (VarLad) [reply Project.toml]: Okay!
Any other change?

Comment (VarLad) [reply Project.toml]: Changed it! Still works!!!😀

Comment (VarLad) [reply Project.toml]: @SimonDanisch Its really odd that I just changed Julia versions and it works...🙂
Any idea, as to why thats the case?

File: Project.toml [hunk 2]
```diff
@@ -28,8 +28,8 @@ ImageMagick = "1.1"
 JSServe = "1.2"
 Observables = "0.3"
 ShaderAbstractions = "0.2.1"
-StaticArrays = "0.12, 1.0"
-julia = "1.3"
+StaticArrays = "0.12, 1.0.2"
```
Comment (SebastianM-C) [inline Project.toml]: Why did you specify the patch version in `1.0.2`? That will force compatibility only with that patch release from the `1.0.x` series (which I don't think it's a good idea).

File: Project.toml [hunk 3]
```diff
@@ -28,8 +28,8 @@ ImageMagick = "1.1"
...
+julia = "1.5, 1.6"
```
Comment (SebastianM-C) [inline Project.toml]: Was the support for older Julia versions dropped?

**Review by SebastianM-C — CHANGES_REQUESTED** [review]
Why did you need to specify the patch version for `StaticArrays`?

File: Project.toml (hunk 3 above)
Comment (VarLad) [reply Project.toml]: @SebastianM-C I think the only problem here was with supporting Julia 1.3. 1.4 looks fine, so do 1.5 and 1.6
After 1.6 is here, I doubt anyone will use 1.5.x or older🙂
I also hear, 1.7 will be the new LTS

File: Project.toml (hunk 2 above)
Comment (VarLad) [reply Project.toml]: I've changed it again and it works.

File: Project.toml (hunk 3 above)
Comment (VarLad) [reply Project.toml]: @SebastianM-C 
Currently, my only change is dropping the 1.3 support
I don't know why but it downgrades a lot of stuff

Comment (VarLad) [reply Project.toml]: Any other problem?

**Review by SebastianM-C — APPROVED** [review]
I'm not sure why julia 1.3 causes problems, but I think this should be fine.

**Comment by VarLad** [comment]
I don't know why, but WGLMakie just installs without downgrading anything...
And I didn't change anything.... 
Not sure whats going on here, but since I don't have any problem with WGLMakie's official 0.3.2 release, I'm closing this PR

PS: Sorry for the trouble! Julia works in mysterious ways🙂

**Pull request closed without merging**